bots/engine.py

```diff
@@ -85,17 +85,19 @@ def start():
         botsglobal.logger = botsinit.initenginelogging(process_name)
         atexit.register(logging.shutdown)
     except Exception as msg:
+        print('Error initialising logging:' + msg)
```

Comment:
not needed, is via logger

Reply:
but this error is during log init, so never gets logged!
(I had this error a long time ago and took me a while to track down what caused it... but can't remember what it was now. Maybe permissions on the log folder)